custom_components/bond/light.py

```diff
@@ -1,9 +1,17 @@
 """Bond Home Light Integration"""
 from homeassistant.components.light import (
-    ATTR_BRIGHTNESS, PLATFORM_SCHEMA, Light)
+    ATTR_BRIGHTNESS,PLATFORM_SCHEMA,Light)
```

Comment:
You can also check if the device supports the actions
```
SetBrightness
SetUpLightBrightness
SetDownLightBrightness
```
and then add this feature to the light